.github/workflows/ci.yml

```diff
@@ -40,7 +40,27 @@ jobs:
       - name: Run lint script
         run: npm run lint
 
-  test:
+  lint-datalock-chaincode:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run lint script
+        uses: golangci/golangci-lint-action@v2
+        with:
+          working-directory: ./utility-emissions-channel/chaincode/datalock
+
+  test-datalock-chaincode:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./utility-emissions-channel/chaincode/datalock
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
```

Comment:
Specify the version of go from [go.mod](https://github.com/hyperledger-labs/blockchain-carbon-accounting/blob/8695974c8c088a9dd883511ec16dcd0b35077340/utility-emissions-channel/chaincode/datalock/go.mod#L3) for all tests to run and pass